cmd/manager/main.go

```diff
@@ -76,6 +68,22 @@ func main() {
 
 	pflag.Parse()
 
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
+		default:
+			fmt.Printf("unknown sub-command: %v\n", os.Args[1])
+			fmt.Print("available sub-commands:\n\tversion\n")
+			fmt.Print("available flags:\n")
+			pflag.PrintDefaults()
+			os.Exit(64)
```

Comment:
Please stick to `os.Exit(1)` as being done everywhere else in `main()`. There is no reason to introduce a user defined exit number.

Reply:
exit code 64 is for `command line usage error`. that's why went with 64. But a generic exit code `1` should work.

Reply:
That is for system programs from a sysexit perspective and is typically for C/C++ programs. It need not apply to all application programs.